glance.html

```diff
@@ -279,7 +279,7 @@ <h2>Acknowledgments</h2>
 			<p>Additional information about participation in the Sustainable Web Interest Group can be found <a href="https://github.com/w3c/sustainableweb-ig/">within the GitHub repository</a> of the Interest Group.</p>
 			<section> <!-- Draft Contributors -->
 				<h2>Participants active in the development of this document</h2>
-				<p>Alexander Dawson, Andy Blum, Chris Wilson, Michelle Barker, Neil Clark, Tim Frick</p>
+				<p>Alexander Dawson, Andy Blum, Chris Wilson, Florian Rivoal, Hidde De Vries, Michelle Barker, Neil Clark, Tantek Çelik, Tim Frick</p>
```

Comment:
small nit, this lowercases the `d` in my name: 

```suggestion
				<p>Alexander Dawson, Andy Blum, Chris Wilson, Florian Rivoal, Hidde de Vries, Michelle Barker, Neil Clark, Tantek Çelik, Tim Frick</p>
```